webcompat/views.py

```diff
@@ -278,6 +295,24 @@ def jumpship(e):
     return redirect(url_for('index'))
 
 
+@app.errorhandler(400)
+def unauthorized(err):
```

Comment:
Ah my bad!!!!
Need better eyeballs.

@miketaylr The function is being redefined You need to change the name when you cut and paste :p